Apps/W1/File - Azure BLOB Storage Connector/app/src/BlobStorageAccount.Page.al

```diff
@@ -0,0 +1,84 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Displays an account that was registered via the Blob Storage connector.
+/// </summary>
+page 80100 "Blob Storage Account"
+{
+    SourceTable = "Blob Storage Account";
+    Caption = 'Azure Blob Storage Account';
+    Permissions = tabledata "Blob Storage Account" = rimd;
+    PageType = Card;
+    Extensible = false;
+    InsertAllowed = false;
+    DataCaptionExpression = Rec.Name;
+
+    layout
+    {
+        area(Content)
+        {
+            field(NameField; Rec.Name)
+            {
+                ApplicationArea = All;
+                Caption = 'Account Name';
+                ToolTip = 'Specifies the name of the Storage account connection.';
+                ShowMandatory = true;
+                NotBlank = true;
+            }
+
+            field(StorageAccountNameField; Rec."Storage Account Name")
+            {
+                ApplicationArea = All;
+                Caption = 'Storage Account Name';
+                ToolTip = 'Specifies the Azure Storage name.';
+            }
+
+            field(Password; Password)
```

Comment:
I know taht currently only shared key is used, but I can imagine, that there might be more options to authenticate in the future.
Maybe we should already prepare this.
It's not a must, but we should at least think about this.

Reply:
I think it is easy to implement new auth types without being destructive.